src/main/java/org/opensearch/flowframework/transport/DeleteWorkflowTransportAction.java

```diff
@@ -64,6 +64,15 @@ protected void doExecute(Task task, WorkflowRequest request, ActionListener<Dele
             ThreadContext.StoredContext context = client.threadPool().getThreadContext().stashContext();
             logger.info("Deleting workflow doc: {}", workflowId);
             client.delete(deleteRequest, ActionListener.runBefore(listener, context::restore));
+
+            ActionListener<DeleteResponse> stateListener = ActionListener.wrap(response -> {
+                logger.info("Deleted workflow state doc: {}", workflowId);
+            }, exception -> { logger.info("Failed to delet workflow state doc: {}", workflowId, exception); });
```

Comment:
```suggestion
            }, exception -> { logger.info("Failed to delete workflow state doc: {}", workflowId, exception); });
```

Reply:
Couldn't have found that on the first review, eh? ;)

I like red squigglies in my IDE.  (And in Github under squigglies)